packages/components/src/navigation/index.js

```diff
@@ -67,6 +67,14 @@ const Navigation = ( { activeItemId, children, data, rootTitle } ) => {
 		}
 	}, [ activeItem ] );
 
+	const isMounted = useRef( true );
```

Comment:
Sorry, I gave the wrong naming before.  I think this should be `isMounted = useRef( false );` and then update to `true` to semantically match `isMounted`.

Otherwise we can update the name to `isMounting` to be semantically correct without changing the other logic.

Reply:
Good idea, thanks. Fixed in https://github.com/WordPress/gutenberg/pull/24960/commits/fdad0cad4b79d381dc47bb8b378163e804622169